ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -1,119 +1,222 @@
 #!nsh
 #
-# Standard startup script for sensor drivers.
+# Sensor driver startup script.
 #
+# NOTE: environment variable references:
+#    If the dollar sign ('$') is followed by a left bracket ('{') then the
+#    variable name is terminated with the right bracket character ('}').
+#    Otherwise, the variable name goes to the end of the argument.
+#
+#
+# NOTE: COMMENT LINES ARE REMOVED BEFORE STORED IN ROMFS.
+#
+
+#
+# Declare local script variables and initialize default values.
+#
+set BOARD_FMUV3    false
+
+#
+# Setup for generic configurations.
+#
+
+# Configure all I2C buses to 100 KHz as they are all external or slow.
+fmu i2c 1 100000
+fmu i2c 2 100000
+
+# ADC.
+adc start
```

Comment:
These were not executed on AEROFC_V1 (it uses a different adc for example)

Reply:
Agreed.  Please see comment below.